.bsp/sbt.json

```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.4.4","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/geoffwatson/Library/Application Support/JetBrains/IntelliJIdea2020.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
\ No newline at end of file
+{"name":"sbt","version":"1.4.4","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/usr/local/Cellar/sbt/1.1.5/libexec/bin/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
Did you intend to include this change? What does it do?

Reply:
no idea where that came from.. removed, thx